package/MDAnalysis/lib/util.py

```diff
@@ -1164,32 +1164,3 @@ def wrapper(self, *args, **kwargs):
         return wrapper
 
     return cached_lookup
-
-
-def unique_rows(arr):
```

Comment:
Haha, PyCharm was telling me I deleted the only call made to it in the project (except in the testsuite). Should I un-delete it?